common/common/src/main/java/io/helidon/common/FeatureCatalog.java

```diff
@@ -191,6 +191,14 @@
                     .path("HCP Vault")
                     .flavor(HelidonFlavor.SE)
                     .experimental(true));
+        add("io.helidon.integrations.microstream",
+            FeatureDescriptor.builder()
+                    .name("Microstream")
+                    .description("Microstream Integration")
+                    .path("Microstream")
+                    .flavor(HelidonFlavor.SE)
+                    .experimental(true)
+                    .nativeSupported(false));
```

Comment:
Actually I have tried to build examples with native for the demo long time ago. I stopped working?